docs/en/ingest-management/elastic-agent/configuration/inputs/input-configuration.asciidoc

```diff
@@ -11,34 +11,35 @@ By default {agent} collects system metrics, such as CPU, memory, network, and fi
 
 ["source","yaml"]
 -----------------------------------------------------------------------
-- id: unique-system-metrics-id <1>
-  type: system/metrics <2>
-  use_output: default <3>
+- type: system/metrics <1>
+  id: unique-system-metrics-id <2>
+  data_stream.namespace: default <3>
+  use_output: default <4>
   meta:
-    package: <4>
+    package: <5>
       name: system
       version: 0.10.9
```

Comment:
This `meta:` section doesn't appear in `elastic-agent.yml` so perhaps we should remove it from the docs?